cmake/multi_image.cmake

```
@@ -335,6 +335,10 @@ function(add_child_image_from_source)
       # Place the result in the CMake cache and remove local scoped variable.
       foreach(file CONF_FILE DTC_OVERLAY_FILE)
         if(DEFINED ${ACI_NAME}_${file})
+          if(DEFINED ${ACI_NAME}_FILE_SUFFIX)
+            zephyr_file_suffix(${ACI_NAME}_${file} SUFFIX ${${ACI_NAME}_FILE_SUFFIX})
+          endif()
```

Not needed, explained in other comment:
```suggestion
          zephyr_file_suffix(${ACI_NAME}_${file} SUFFIX ${${ACI_NAME}_FILE_SUFFIX})
```

Second observation, why doesn't the call above include `SUFFIX` in the `ncs_file()` call instead of this code ?: 
https://github.com/gmarull/sdk-nrf/blob/0147d0e832c90c46b3cfb134a2acb0f5877c8903/cmake/multi_image.cmake#L327-334

---

so should be fixed at the correct location

---

No, the `ncs_file()` is an extension to `zephyr_file()`, so if zephyr_file() handles SUFFIX, so should ncs_file().
https://github.com/gmarull/sdk-nrf/blob/51e583bae91e95306c37b907d1b865c0d7f26ffb/cmake/extensions.cmake#L137-L145